MAINTAINERS.MD

```diff
@@ -1,3 +1,4 @@
 | Org                    | Name                                      | GitHub Username        |
 | -----------------------| ------------------------------------------| -----------------------|
-| TIM                    | Fabrizio Moggio                           | FabrizioMoggio         |
+| TIM                    | Fabrizio Moggio                           | FabrizioMoggio         
+|T - Systems             | Kanhaiya Sharma                           | kans-sharma            |
```

Comment:
Kanhaiya Sharma will support the work on the API for Deutsche Telekom:

```suggestion
| Deutsche Telekom AG             | Kanhaiya Sharma                           | kans-sharma            |
```

Reply:
good